ui/app/AppLayouts/Profile/Sections/Ens/ENSPopup.qml

```diff
@@ -21,8 +21,12 @@ ModalPopup {
 
     StyledText {
         id: lbl1
-        //% "Your messages are displayed to others with this username:"
-        text: qsTrId("your-messages-are-displayed-to-others-with-this-username-")
+        
+        text: profileModel.ens.preferredUsername ? 
+              //% "Your messages are displayed to others with this username:"
+              qsTrId("your-messages-are-displayed-to-others-with-this-username-")
+              :
+              qsTr("Once you select an username, you won’t be able to disable it afterwards, only choose a different username to display.")
```

Comment:
I'm not sure why, but `username` doesn't come with an `an` but an `a`
I also changed the other sentence because it was way too convoluted
```suggestion
              qsTr("Once you select a username, you won’t be able to disable it afterwards. You will only be able choose a different username to display.")
```